### PR TITLE
PMT Buddy

### PR DESCRIPTION
**Pull request by realooffyy**
# BrowserBuddy Extension Submission

## Project Details

1. **What is the name of your project?**
   PMT Buddy

3. **How many hours did you spend developing this?**  
   About 20 hours

4. **What problem does your extension solve, or what makes it unique?**  
   This extension contains features intended to improve your experience on revision websites, by removing promotional content and distractions, or adding elements, to let you focus on what you're really there for!

5. **How many people have tested your extension?**  
   I have had 7 of my friends to test before publishing and now have 5 Chromium-based users.

6. **Does your extension work in both Chrome and Firefox?**
   Yes, despite the warnings provided by the browser.

---

## Additional Comments

https://chromewebstore.google.com/detail/pmt-buddy/moepnndepfamgglegngndbagiobmhpop

### CODE REVIEW

**Comment by JosiasAurel** [comment]
Can you link some websites where I can test this extension?

**Comment by realooffyy** [comment]
Sure! These are some examples of applications for the extension:
- [Physics and Maths Tutor](https://www.physicsandmathstutor.com/) (Hide paid tutor content)
- [SaveMyExams](https://www.savemyexams.com/a-level/maths_pure/aqa/18/revision-notes/1-proof/1-1-proof/1-1-1-language-of-proof/) (the unlocker is only effective after you've viewed your weekly 5 free revision notes)
- [studocu](https://www.studocu.com/en-gb/document/ark-academy/physics/suvat-and-motion-graphs/88559602?shared=u)

The domains are also linked in the README.

File: submissions/PMT Buddy/build/background.js
Comment (JosiasAurel) [inline submissions/PMT Buddy/build/background.js:1]: please remove the contents of the build folder

**Review by JosiasAurel — CHANGES_REQUESTED** [review]
This extension seems to limited in terms of the number of websites where it works.

**Comment by realooffyy** [comment]
> This extension seems to limited in terms of the number of websites where it works.

Yes, this is intentional. I created this extension to help remove/reduce the censorship of and difficulty in accessing otherwise free resources online.

**Comment by JosiasAurel** [comment]
I still find paid content on the first website 
image

image

**Comment by realooffyy** [comment]
That's odd, could you make sure the settings are enabled in the extension popup? Refreshing the page should apply changes.

**Comment by realooffyy** [comment]
As for studocu, I left that message over the blurred page as the website doesn't send the unblurred version. That extension feature is actually hiding this banner:

image

**Comment by JosiasAurel** [comment]
I've tried reloading the page but that didn't work. It seems to be removing useful information (advertising their tutoring services - which will be potentially useful - on a free tutoring site seems fair to me), and removing a banner that provides information on how to unlock all the content seems counterintuitive if you want to have full access.

We're also past deadline so I'll be rejecting this submission.

**Pull request closed without merging**